renderer.js

```diff
@@ -1,3 +1,6 @@
 // This file is required by the index.html file and will
 // be executed in the renderer process for that window.
-// All of the Node.js APIs are available in this process.
+// No Node.js APIs are available in this process because
+// `nodeIntegration` is turned off. Use `preload.js` to
```

Comment:
I would instead perhaps say:
```
Node.js APIs are not available in this process by default. To make them
available, either set `nodeIntegration` to `true` in your BrowserWindow's 
webPreferences or use `preload.js` to selectively enable features 
needed in the renderer process.
```

Reply:
At the risk of bikeshedding -- this is going to be the first Electron app that a new dev sees, so IMO we should try to concisely explain the choice involved here. I'd go for something more like

```js
// For security, Node.js is inaccessible by default to the renderer.
// To use Node.js, either add your code to `preload.js` (preferred) or
// set `nodeIntegration` to `true` when creating your BrowserWindow.
// See https://electronjs.org/docs/tutorial/security for more information.
```